src/status_im2/contexts/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.activity-center.notification.mentions.view
+  (:require [clojure.string :as string]
+            [i18n.i18n :as i18n]
+            [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [status-im.utils.datetime :as datetime]
+            [status-im2.contexts.activity-center.notification.common.view :as common]
+            [status-im2.contexts.activity-center.notification.mentions.style :as style]
+            [utils.re-frame :as rf]))
+
+(defn message-body [message]
+  (let [parsed-text          (get-in message [:content :parsed-text])
+        parsed-text-children (:children (first parsed-text))]
+    (into [quo/text {:number-of-lines     2
+                     :style               style/tag-text
+                     :accessibility-label :activity-message-body
+                     :size                :paragraph-1}]
+          (map-indexed (fn [index {:keys [type literal]}]
+                         ^{:key index}
+                         (case type
+                           "mention" [quo/text  {:style style/mention-text
+                                                 :size  :paragraph-1}
+                                      (str "@" (rf/sub [:contacts/contact-name-by-identity literal]))]
+                           literal)) parsed-text-children))))
+
+(defn view
```

Comment:
i believe we decided to name views with meaningful names, like `notification-item` etc, not view

Reply:
yeah, then all my view files screwed in status-im2 :) we need this in guidelines for sure

Reply:
@J-Son89 do you remember what was the agreement?

Reply:
imo `mentions/view` looks better than `mentions/mentions`

Reply:
cc @siddarthkay pls don't rename methods in your PR

Reply:
@siddarthkay but it should be view namespace and method, without `s` on the end

Reply:
> cc @siddarthkay pls don't rename methods in your PR

too late @flexsurfer

jk I will revert the renaming